src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,212 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String createRequest = "INSERT INTO cars (model, manufacturer_id)"
+                + " VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement createCarStatement =
+                         connection.prepareStatement(createRequest,
+                                 Statement.RETURN_GENERATED_KEYS)) {
+            createCarStatement.setString(1, car.getModel());
+            createCarStatement.setLong(2, car.getManufacturer().getId());
+            createCarStatement.executeUpdate();
+            ResultSet generatedKeys = createCarStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                car.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert car to DB " + car, e);
+        }
+        insertDrivers(car);
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String selectRequest = "SELECT c.id as car_id, model, "
+                + "m.id as manufacturer_id, m.name, m.country "
+                + "FROM CARS c JOIN manufacturers m "
+                + "ON c.manufacturer_id = m.id WHERE c.id = ? and c.is_deleted = false;";
```

Comment:
```suggestion
                + "ON c.manufacturer_id = m.id WHERE c.id = ? and c.is_deleted = FALSE;";
```
if you use upper case, use it everywhere, the same for other queries